services/resource-usage-tracker/tests/unit/with_dbs/test_api_resource_tracker_service_runs__export.py

```diff
@@ -0,0 +1,70 @@
+from unittest.mock import Mock
+
+import pytest
+import sqlalchemy as sa
+from moto.server import ThreadedMotoServer
+from pydantic import AnyUrl, parse_obj_as
+from pytest_mock import MockerFixture
+from pytest_simcore.helpers.typing_env import EnvVarsDict
+from servicelib.rabbitmq import RabbitMQRPCClient
+from servicelib.rabbitmq.rpc_interfaces.resource_usage_tracker import service_runs
+
+pytest_simcore_core_services_selection = [
+    "postgres",
+    "rabbit",
+]
+pytest_simcore_ops_services_selection = [
+    "adminer",
+]
+
+_USER_ID = 1
+
+
+@pytest.fixture
+def enable_resource_usage_tracker_s3(monkeypatch):
+    # Set the environment variable
+    monkeypatch.setenv("RESOURCE_USAGE_TRACKER_S3_ENABLED", "1")
+
+
+@pytest.fixture
+async def mocked_export(mocker: MockerFixture):
+    mock_export = mocker.patch(
+        "simcore_service_resource_usage_tracker.services.resource_tracker_service_runs.ResourceTrackerRepository.export_service_runs_table_to_s3",
+        autospec=True,
+    )
+
+    return mock_export
+
+
+@pytest.fixture
+async def mocked_presigned_link(mocker: MockerFixture):
+    mock_presigned_link = mocker.patch(
+        "simcore_service_resource_usage_tracker.services.resource_tracker_service_runs.SimcoreS3API.create_presigned_download_link",
+        return_value=parse_obj_as(
+            AnyUrl,
+            "https://www.testing.com/",
+        ),
+    )
+
+    return mock_presigned_link
+
+
+@pytest.mark.rpc_test()
```

Comment:
this is a new mark. Any particular reason for it?

Reply:
As currently I am supporting for some functionality both rpc and rest endpoints, this is just for my convenient that at some point I am planning to remove the rest endpoints with it tests, so when I have duplicated tests for both cases this is a way how I differentiate between them. Look for example here: `osparc-simcore/services/resource-usage-tracker/tests/unit/with_dbs/test_api_resource_tracker_service_runs__list_billable.py`